### DISCUSSION TITLE
Include node_module module

### DISCUSSION THREAD

**Discussion by peteringram0** [TypeStrong/ts-node#1544] [Q&A] [answered]
By default ts-node skips everything inside node_modules as per (https://github.com/TypeStrong/ts-node#skipping-node_modules). Is there a flag or a way to include a specific module to be included?

**Comment by cspotcode** [comment, marked as answer]
You can try fiddling with the `ignore` option.  UX here isn't great -- it's interpreted as a regular expression.  Probably best to declare it in your tsconfig.json file.

Docs links:
https://typestrong.org/ts-node/docs/options#transpilation
https://typestrong.org/ts-node/docs/configuration#via-tsconfigjson-recommended

Setting the `TS_NODE_DEBUG` environment variable to `true` *might* give you some useful logging about what is and is not being ignored, but I haven't looked at the debug logging in a while so I don't remember for certain.

**Reply by peteringram0** [reply]
Thanks for your help!! I did try --ignore before however didn't release it was a regex. I got it working in the end using

````typescript
--ignore '/node_modules/(?!package-name/.*)'
````
